### PR TITLE
220 abx project templates must be updated

### PR DESCRIPTION
**Pull request by mtopchieva**
### Description
Updated the package.json template for generating abx actions so that it contains more of the recently implemented parameters: base, memoryLimitMb, timeoutSec, provider and abx (inputSecrets, inputConstants, etc.)

### Checklist
- [x] I have updated CHANGELOG.md with a short summary of the changes introduced

### Testing
Updated the package.json template for generating abx actions so that it contains more of the recently implemented parameters.

### Release Notes
Updated the package.json template for generating abx actions so that it contains the parameters: base, memoryLimitMb, timeoutSec, provider and abx (inputSecrets, inputConstants, etc.)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+### Enhancements
+* [artifact-manager] Issue 220 / mvn archetype:generate for abx project - updated the package.json template for generating abx actions
```
Comment (VenelinBakalov) [inline CHANGELOG.md:2]: Thanks for updating this! Can you also add an update to the RELEASE.md file

Comment (mtopchieva) [reply CHANGELOG.md:2]: Done

**Comment by VenelinBakalov** [comment]
@mtopchieva sorry for the delay. could you update the remaining merge conflicts and then we can merge this as well

**Review by Michaelpalacce — APPROVED** [review]
lgtm

**Pull request merged**